assets/scss/tools/_string.scss

```diff
@@ -8,3 +8,66 @@
 
     @return $string;
 }
+
+// https://stackoverflow.com/questions/32376461/how-to-split-a-string-into-two-lists-of-numbers-in-sass
+@function str-split($string, $separator) {
+
+   $index : str-index($string,  $separator);
+
+   $str-1 : str-slice($string, 1, $index - 1);
+   $str-2 : str-slice($string, $index + 1);
+
+   @return $str-1 $str-2;
+}
+
+// https://www.sassmeister.com/gist/9fa19d254864f33d4a80
+@function to-number($value) {
```

Comment:
Since we are adding all these helpers should we think about adding some unit tests to go with it as well. This is just for the team to think over. Not a priority for this PR @bigcommerce/stencil-team